test/test_dependency_versions.py

```diff
@@ -21,12 +50,13 @@ def get_versions_from_constraint_file(constraint_file):
     return versions
 
 
-@pytest.mark.parametrize('python_version', ['2.7', '3.5', '3.8'])
-def test_pinned_versions(python_version):
+@pytest.mark.parametrize('python_version', ['2.7', '3.5', '3.6', '3.8'])
```

Comment:
Where do we get these versions, and why isn't 3.7 there?

Reply:
The python-specific constraints files are defined in https://github.com/joerick/cibuildwheel/blob/master/bin/update_dependencies.py#L22

I also added the latest version we support, to test the fallback constraint file. Python 3.7 was happy with the fallback one, so it doesn't have a specific constraint file.

Reply:
Aaaaah, I see. Then it's the `3.8` that's actually confusing me, because I kind of knew about the fallback setup, but then 3.8 wouldn't make sense :-) Thanks!